package.json

```diff
@@ -1,84 +1,84 @@
 {
-    "name": "make-sense",
-    "version": "1.0.0",
-    "private": true,
-    "dependencies": {
-        "@material-ui/core": "^4.12.3",
-        "@tensorflow-models/coco-ssd": "^2.2.2",
-        "@tensorflow-models/posenet": "^2.2.2",
-        "@tensorflow/tfjs": "^3.8.0",
-        "@tensorflow/tfjs-backend-cpu": "^3.9.0",
-        "@tensorflow/tfjs-backend-webgl": "^3.9.0",
-        "@tensorflow/tfjs-core": "^3.9.0",
-        "@tensorflow/tfjs-node": "^3.9.0",
-        "@types/jest": "27.0.1",
-        "@types/node": "16.7.6",
-        "@types/react": "17.0.19",
-        "@types/react-dom": "17.0.9",
-        "axios": "^0.27.1",
-        "classnames": "^2.3.1",
-        "file-saver": "^2.0.5",
-        "jszip": "^3.7.1",
-        "lodash": "^4.17.21",
-        "mobile-detect": "^1.4.5",
-        "moment": "^2.29.1",
-        "node-sass": "^6.0.1",
-        "react": "^16.8.6",
-        "react-custom-scrollbars": "^4.2.1",
-        "react-dom": "^17.0.2",
-        "react-dropzone": "^11.3.4",
-        "react-modal-resizable-draggable": "^0.1.6",
-        "react-redux": "^7.2.4",
-        "react-scripts": "^4.0.3",
-        "react-select": "^5.3.0",
-        "react-spinners": "^0.11.0",
-        "redux": "^4.1.1",
-        "redux-thunk": "^2.4.1",
-        "typescript": "4.4.2",
-        "uuid": "^8.3.2"
-    },
-    "scripts": {
-        "start": "react-scripts start",
-        "build": "react-scripts build",
-        "test": "react-scripts test",
-        "test:coverage": "npm test -- --coverage",
-        "eject": "react-scripts eject"
-    },
-    "eslintConfig": {
-        "plugins": [
-            "react-hooks"
-        ],
-        "extends": [
-            "react-app",
-            "react-app/jest"
-        ],
-        "rules": {
-            "react-hooks/rules-of-hooks": "error",
-            "react-hooks/exhaustive-deps": "warn"
-        }
-    },
-    "browserslist": {
-        "production": [
-            ">0.2%",
-            "not dead",
-            "not op_mini all"
-        ],
-        "development": [
-            "last 1 chrome version",
-            "last 1 firefox version",
-            "last 1 safari version"
-        ]
-    },
-    "devDependencies": {
-        "@redux-devtools/core": "^3.9.0",
-        "@types/file-saver": "^2.0.3",
-        "@types/lodash": "^4.14.172",
-        "@types/react-custom-scrollbars": "^4.0.8",
-        "@types/react-redux": "^7.1.18",
-        "@types/uuid": "^8.3.1",
-        "eslint-plugin-react-hooks": "^4.4.0",
-        "prettier": "^2.3.2",
-        "tslint": "^6.1.3",
-        "tslint-react": "^5.0.0"
-    }
-}
+  "name": "make-sense",
```

Comment:
Please follow the current code format. Please do not change codes not related to this PR.

Reply:
yeah I will change it imedialy